tpot/tpot.py

```diff
@@ -496,6 +500,181 @@ def _random_forest(self, input_df, max_features):
         return self._train_model_and_predict(input_df, RandomForestClassifier, n_estimators=500,
                                              max_features=max_features, random_state=42, n_jobs=-1)
 
+    def _ada_boost(self, input_df, learning_rate):
+        """Fits an AdaBoost classifier
+
+        Parameters
+        ----------
+        input_df: pandas.DataFrame {n_samples, n_features+['class', 'group', 'guess']}
+            Input DataFrame for fitting the classifier
+        learning_rate: float
+            Learning rate shrinks the contribution of each classifier by learning_rate.
+
+        Returns
+        -------
+        input_df: pandas.DataFrame {n_samples, n_features+['guess', 'group', 'class', 'SyntheticFeature']}
+            Returns a modified input DataFrame with the guess column updated according to the classifier's predictions.
+            Also adds the classifiers's predictions as a 'SyntheticFeature' column.
+
+        """
+        return self._train_model_and_predict(input_df, AdaBoostClassifier,
```

Comment:
Check `_logistic_regression` for an example: https://github.com/rhiever/tpot/blob/master/tpot/tpot.py#L516

```python
C = max(0.0001, C)
```

0.0001 seems like a fine minimum value for now.